gozstd_test.go

```diff
@@ -54,6 +55,22 @@ func TestDecompressSmallBlockWithoutSingleSegmentFlag(t *testing.T) {
 	})
 }
 
+func TestCompressEmpty(t *testing.T) {
+	var dst [64]byte
+	res := Compress(dst[:0], nil)
+	if len(res) > 0 {
+		t.Fatalf("unexpected non-empty compressed frame: %X", res)
+	}
+}
+
+func TestDecompressTooLarge(t *testing.T) {
+	src := []byte{40, 181, 47, 253, 228, 122, 118, 105, 67, 140, 234, 85, 20, 159, 67}
+	_, err := Decompress(nil, src)
```

Comment:
From the test name I gather that the error here is that the decompressed size is too larger for the dst buf (nil)? It would be a bit easier to read if the dst buf was non-nil, like maybe 1 byte or something